shared/rocroller/lib/source/Graph/HypergraphIncidenceContainer.cpp

```diff
@@ -43,14 +43,21 @@ namespace rocRoller::Graph
 
     void HypergraphIncidenceContainer::deleteTag(int tag)
     {
+        auto srcs = getSrcs(tag);
+        auto dsts = getDsts(tag);
+
         m_incidenceBySrc.erase(tag);
         m_incidenceByDst.erase(tag);
 
-        for(auto& connections : m_incidenceBySrc)
-            std::erase(connections.second, tag);
+        for(auto src : srcs)
+        {
+            AssertFatal(std::erase(m_incidenceBySrc.at(src), tag) > 0);
```

Comment:
Because `AssertFatal` is a macro that we may choose to make into a NOP based on compiler flags in the future, it's best not include code that needs to be executed in the macro.

I've shot myself in the foot a few times like this ;)

```suggestion
            auto erased = std::erase(m_incidenceBySrc.at(src), tag);
            AssertFatal(erased > 0);
```